selfdrive/locationd/paramsd.py

```diff
@@ -102,6 +105,14 @@ def handle_log(self, t, which, msg):
       self.kf.filter.reset_rewind()
 
 
+def check_valid_with_hysteresis(current_valid: bool, val: float, threshold: float = OFFSET_MAX, lowered_threshold: float = OFFSET_LOWERED_MAX):
```

Comment:
i'd remove the defaults here. it's easy to write a bug like that roll threshold one